system/parted.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016, Vladimir Kozhukalov <kozhukalov@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+import locale
+import math
+
+
+DOCUMENTATION = '''
+---
+author: "Vladimir Kozhukalov"
+module: parted
+short_description: Configure block device partitions
+description:
+  - This module creates, removes block device partitions.
+version_added: "2.2"
+options:
+  device:
+    description:
+    - The device file. E.g. /dev/sda
+    required: true
+  align:
+    description:
+    - Alignment type for newly created partitions
+    required: false
+    choices: ["none", "cylinder", "minimal", "optimal"]
```

Comment:
the default doc is missing according your arg_spec:

~~~
default: optimal
~~~